client/my-sites/site-settings/podcasting-details/private-site.tsx

```diff
@@ -3,22 +3,36 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
+import {localize, LocalizeProps} from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import { getSelectedSiteSlug } from 'state/ui/selectors';
 
-function PodcastingPrivateSiteMessage( { siteSlug, translate } ) {
+interface Props extends LocalizeProps {
+  siteSlug: string,
+	isComingSoon: boolean,
+}
+
+const PodcastingPrivateSiteMessage: React.FC<Props> = function PodcastingPrivateSiteMessage ( { siteSlug, translate, isComingSoon }: Props ) {
 	return (
 		<div className="podcasting-details__private-site">
 			<p>
-				{ translate( "This site's visibility is currently set to {{strong}}Private{{/strong}}.", {
-					components: { strong: <strong /> },
-					comment:
-						'The translation for "Private" should match the string on the Settings > General page.',
-				} ) }
+				{ isComingSoon ? (
+					translate( "This site's visibility is currently set to {{strong}}Coming Soon{{/strong}}.",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'This site's visibility is currently set to {{strong}}Private{{/strong}}.' )` *ES Score: 6*